core/components/atoms/text/__stories__/variants/Appearance.story.jsx

```diff
@@ -1,3 +1,4 @@
+import classNames from 'classnames';
```

Comment:
I think we can not use the `classNames` utility in stories, please verify and remove it.

Reply:
It has not been used anywhere but this works fine still should I write it making the backgroundClass a variable and using ternary operator?

Reply:
So since we don't expose this utility we should not provide this in the example as copying our example code directly will break the apps. Feel free to use the ternary operator or any better way of doing it.